Remotion/Web/UnitTests/Core/Services/WebServiceFactoryTests/CreateScriptService.cs

```diff
@@ -42,14 +42,45 @@ private interface IValidScriptService
       void MethodWithParameters (int param1, bool param2);
     }
 
+    private interface IValidInheritedScriptServiceWithMethodsOnBase : IValidScriptService
+    {
+    }
+
+    private interface IValidInheritedScriptServiceWithMethodsOnBaseAndDerived : IValidScriptService
+    {
+      void InheritedScriptMethod ();
+    }
+
+    private interface IValidInheritedScriptServiceWithMethodsOnMultipleBases : IValidInheritedScriptServiceWithMethodsOnBaseAndDerived
+    {
+    }
+
     private interface IInvalidScriptServiceWithMissingScriptMethodAttribute
     {
       void MethodWithoutScriptMethodAttribute ();
     }
 
+    private interface IInvalidInheritedScriptServiceWithMethodsOnBase : IInvalidScriptServiceWithMissingScriptMethodAttribute
+    {
+    }
+
+    private interface IInvalidInheritedScriptServiceWithMethodsOnBaseAndDerived : IInvalidScriptServiceWithMissingScriptMethodAttribute
+    {
+      void InheritedScriptMethodWithoutScriptMethodAttribute ();
+    }
+
+    private interface IInvalidInheritedScriptServiceWithMethodsOnMultipleBases : IInvalidInheritedScriptServiceWithMethodsOnBase
```

Comment:
Oh, okay, we've got a mis-communication here: I meant multiple-interitance, not a nested hierarchy.